README.md

```diff
@@ -5,9 +5,9 @@ Once a termination notice is received, it will try to gracefully stop all the po
 
 ### Helm
 
-A helm chart has been created for this tool, and at time of writing was in the `incubator` repository.
+A helm chart has been created for this tool, and at time of writing was in the `stable` repository.
 
-    $ helm install incubator/kube-spot-termination-notice-handler
+    $ helm install stable/kube-spot-termination-notice-handler
```

Comment:
The name of the chart is `stable/k8s-spot-termination-handler` - would you mind updating?

Reply:
Updated